cypress/integration/systemIntake.spec.js

```diff
@@ -0,0 +1,136 @@
+import cmsGovernanceTeams from '../../src/constants/enums/cmsGovernanceTeams';
+
+describe('The System Intake Form', () => {
+  before(() => {
+    // TODO: We shouldn't be logging in for every test suite.
+    // We should find a way to login programmatically.
+    cy.visit('/login');
+
+    cy.get('#okta-signin-username').type(Cypress.env('username'));
+    cy.get('#okta-signin-password').type(Cypress.env('password'));
+    cy.get('#okta-signin-submit').click();
+    cy.task('generateOTP', Cypress.env('otpSecret')).then(token => {
+      cy.get('input[type="tel"]').type(token);
+    });
+    cy.get('input[type="submit"').click();
+    cy.location('pathname', { timeout: 20000 }).should('equal', '/');
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
```

Comment:
restoring okta stuff in local storage and navigating to the system intake before every test.